script/refunder.js

```diff
@@ -2,36 +2,52 @@ const ethers = require("ethers");
 const ABI = require("../abi/ethflow.json");
 require("dotenv").config();
 
+const CREATE_ORDER_SELECTOR = "322bba21";
+const ETHFLOW_ORDER_LENGTH = 576;
+
 async function main() {
-  const provider = new ethers.providers.JsonRpcProvider(
-    process.env.NODE_URL,
-  );
+  const provider = new ethers.providers.JsonRpcProvider(process.env.NODE_URL);
   const tx_hash =
     process.env.ETHFLOW_TX_HASH ||
     "0x1416bc69abce952dc42578ea5bbeacd6dbbf15130d30d6305a686a2fb5a6690f";
   const tx = await provider.getTransaction(tx_hash);
   const receipt = await tx.wait();
   const iface = new ethers.utils.Interface(ABI);
-  const log = receipt.logs[0];
+  const order_placement_event_hash =
+    "0xcf5f9de2984132265203b5c335b25727702ca77262ff622e136baa7362bf1da9";
+  const logs = receipt.logs.filter(
+    (log) =>
+      // topic0 is required by the iface.parseLog() function
```

Comment:
This comment is odd. Ultimately we are interested in finding a specific event and parsing its data.